### ISSUE TITLE
11.0.0: no-setup-in-describe doesn't work for any extra calls after a call to a suite config expression (timeout, slow, retries)

### ISSUE THREAD

**Issue by melink14** [lo1tuma/eslint-plugin-mocha#383]
I have code like this:

```ts
describe("some test", function () {
   this.timeout(5);
   ...
```

Even if I update the code with some random function call:
```ts
describe("some test", function () {
   this.timeout(5);
   ...
   someFunctionCall();
```

I get no reported errors. As soon as the timeout is deleted, then all errors show up immediately. Errors are also reported for any calls that happen before the timeout call.

I first noticed this when my eslint disables for generated tests started messing up and I couldnt' figure out why (https://github.com/melink14/rikaikun/pull/2551/files example fix auto applicaiton in a subsequent PR).

Confirmed the issue doesn't exist in 10.5.0.
